include/zephyr/ipc/icmsg.h

```diff
@@ -35,8 +35,10 @@ struct icmsg_data_t {
 	/* Callbacks for an endpoint. */
 	const struct ipc_service_cb *cb;
 	void *ctx;
+	const struct icmsg_config_t *cfg;
 
 	/* General */
+	struct k_work_delayable bond_notify_work;
```

Comment:
I would like simpler: `notify_work` or just `notify`. At this point this is sending signal (notification).

Reply:
OK